### ISSUE TITLE
Support macOS

### ISSUE THREAD

**Issue by matthewbauer** [matthewbauer/nix-bundle#17]
This will be a meta issue on getting macOS/Darwin/XNU support for nix-bundle. AFAICT no reliable way to chroot exists no macOS for a user. Related links:

- proot-me/proot#81
- [osxfuse](https://osxfuse.github.io/)

**Comment by ariutta** [comment]
Would you be able to use any ideas from [Warp](https://github.com/dgiagio/warp)? It "creates self-contained single binary applications ... and is supported on Linux, Windows and macOS".

**Comment by matthewbauer** [comment]
@ariutta Yeah we can't really use Warp besides as maybe an alternative to AppImage/self extracting bash.

**Comment by Thesola10** [comment]
alternatively I wrote a macOS path redirector for use with Nix: [fakedir](https://github.com/nixie-dev/fakedir)

**Comment by HannesGitH** [comment]
I stumbled across https://github.com/niklaskorz/nix-bundle-darwin which also seems promising
